adafruit_funhouse/network.py

```diff
@@ -107,6 +105,14 @@ def _get_mqtt_client(self):
             return self._mqtt_client
         raise RuntimeError("Please initialize MQTT before using")
 
+    def mqtt_loop(self):
+        """Run the MQTT Loop"""
+        try:
+            if self._mqtt_client is not None:
+                self._mqtt_client.loop()
+        except MQTT.MMQTTException:
+            pass
```

Comment:
`except`-`pass` makes any failure invisible, which is hard to debug. Could you print something, or at least return a boolean success/failure?

Reply:
Sure

Reply:
Thanks - I hope it will not flood the output if it retries frequently.